report/sections/1-introduction.tex

```diff
@@ -0,0 +1,31 @@
+\section{Introduction}
+
+TypeScript is a statically typed programming language that is a superset JavaScript (ECMAScript 2015).
```

Comment:
We use these terms first here, so lets add links/footnotes to tge official websites here